### ISSUE TITLE
Image Thumbnail with loading="eager"

### ISSUE THREAD

**Issue by aweichler** [pimcore/pimcore#9577]
It seems that the generation of a thumbnail with attribute loading="eager" is not possible.

```twig
{{ pimcore_image_thumbnail_html(image, 'fullWidthImage', {'imgAttributes': {'class': 'page-hero__image', 'loading': 'eager'}}) }}
```

The result is the following: 
```html
<picture>
   <source srcset="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--64da1dc0--query.webp 1x, /test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--64da1dc0--query@2x.webp 2x" media="(max-width: 79.99999999em)" type="image/webp">
   <source srcset="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--64da1dc0--query.jpg 1x, /test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--64da1dc0--query@2x.jpg 2x" media="(max-width: 79.99999999em)" type="image/jpeg">
   <source srcset="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--0d31dd33--query.webp 1x, /test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--0d31dd33--query@2x.webp 2x" media="(min-width: 80em)" type="image/webp">
   <source srcset="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--0d31dd33--query.jpg 1x, /test/image-thumb__3__fullWidthImage/full-screen-image-placeholder~-~media--0d31dd33--query@2x.jpg 2x" media="(min-width: 80em)" type="image/jpeg">
   <source srcset="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder.webp 1x, /test/image-thumb__3__fullWidthImage/full-screen-image-placeholder@2x.webp 2x" type="image/webp">
   <source srcset="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder.jpg 1x, /test/image-thumb__3__fullWidthImage/full-screen-image-placeholder@2x.jpg 2x" type="image/jpeg">
   <img class="page-hero__image" loading="lazy" src="/test/image-thumb__3__fullWidthImage/full-screen-image-placeholder.jpg" alt="" width="2400" height="1353">
</picture>
```
Unfortunately the attribute is not taken from the options array.

I think the line in the following link is the prblem: [Thumbnail.php](https://github.com/pimcore/pimcore/blob/da0e662962ad7a38630b4e8c56e993e8a671b961/models/Asset/Image/Thumbnail.php#L401)

Am i right or is there another way to get the attribute to the img tag?

**Comment by brusch** [comment]
Fixed by #9579

**Comment by aweichler** [comment]
Thanks for the quick fix :+1:
